GroceryStore/GroceryStore.Constants/ProductCategories.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GroceryStore.Constants
+{
+    public class ProductCategories
+    {
+        public enum Category
```

Comment:
It's great that you understood how to user Enums but in this case, it'd be an overkill.
You can simply use something like this:
`public const string Fish = "Fish";`